src/api/v2/quickPickWizard/AppResourceFilter.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AppResource, AppResourceFilter as AppResourceFilterOptions } from "@microsoft/vscode-azext-utils/hostapi";
+import { Filter } from "../v2AzureResourcesApi";
+
+export class AppResourceFilter implements Filter<AppResource> {
+
+    constructor(private readonly options: AppResourceFilterOptions) { }
+
+    public matches(resource: AppResource): boolean {
```

Comment:
Once https://github.com/microsoft/vscode-azuretools/pull/1191 is merged, we can change the body of this method to 
```typescript
return resource.azureResourceType === this.azureResourceType
```
without having to make changes elsewhere.